day_6/exercises/burrito.js

```diff
@@ -16,12 +16,36 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+  add_topping(topping) {
+    this.toppings.push(topping);
+  }
+  remove_topping(topping) {
+    this.toppings.pop(topping);
+  }
+  change_protein(protein) {
+    this.protein.push(protein);
+    this.protein.shift();
+  }
 };
 
-var dinner = new Burrito("Beans", "Rice", ["cheese", "salsa", "guacamole"]);
+var dinner = new Burrito(["Beans"], "Rice", ["cheese", "salsa", "guacamole"]);
 
 // Call the methods here
 
 console.log(dinner.protein);
 console.log(dinner.base);
 console.log(dinner.toppings);
```

Comment:
I think great job getting this and the below to work, but consider the questions above.